pubsub/google/cloud/pubsub_v1/futures.py

```diff
@@ -0,0 +1,176 @@
+# Copyright 2017, Google Inc. All rights reserved.
```

Comment:
Note: it's not technically required to make this public.

Reply:
I tend to oppose returning instances of private classes to users. (I think @dhermes dislikes that also.)

Reply:
I **really** dislike that.

Reply:
@jonparrott and I discussed offline. I changed the docs to reference `google.api_core.futures.Future`, otherwise this is staying as-is.

Merging once CI goes green.